frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -783,6 +783,12 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
                                 })
                             }}
                             label="Enabled"
+                            disabled={!featureFlag.can_edit}
+                            disabledReason={
```

Comment:
You only need `disabledReason` (ideally the props type would reflect that it's either one or the other)
```suggestion
                            disabledReason={
```